pxemgr/etcd_discovery_handlers.go

```diff
@@ -104,11 +106,28 @@ func (mgr *pxeManagerT) etcdDiscoveryProxyRequest(r *http.Request) (*http.Respon
 	if err != nil {
 		return nil, err
 	}
+	var transport = http.DefaultTransport
+
+	if strings.HasPrefix(mgr.etcdEndpoint, "https") && mgr.etcdCAFile != "" {
+		customCA := x509.NewCertPool()
+
+		pemData, err := ioutil.ReadFile(mgr.etcdCAFile)
+		if err != nil {
+			return nil, errors.New("Unable to read custom CA file: "+err.Error())
```

Comment:
You don't know it yet but as an intro to code conventions `use lowercase`  `unable to read custom...`